src/login.js

```diff
@@ -0,0 +1,87 @@
+import { getEmailErrorMsg, getPasswordErrorMsg, getNicknameErrorMsg } from './validationErrorMessage.js';
```

Comment:
P1:
사용하지 않는 것은 지워주세요.

```suggestion
import { getEmailErrorMsg, getPasswordErrorMsg } from './validationErrorMessage.js';
```